css/src/atomics/ratio.scss

```diff
@@ -0,0 +1,8 @@
+@each $name, $ratio in $ratios {
+	.ratio-#{$name} {
```

Comment:
Been thinking about this naming compared to our standard naming convention. We typically use the full property name, so this should probably be `.aspect-ratio-#{$name}` to keep with convention.

Reply:
This hasn't been changed yet, despite the uses of it being updated.

Reply:
It's resolved